File: single-language-analysis/dataset-files/java/TestSmell126.java
import org.junit.Test;
import static org.junit.Assert.*;

public class TestSmell126 {

    @Test
    public void testIntegerOverflow() {
        try {
            project10.lex("12345678");
            assertFalse(true);
        } catch (Exception e) {
        }
    }
}

File: single-language-analysis/dataset-files/python/test_smell_126.py
# Original URL: https://github.com/mossprescott/pynand/blob/e02f534d7afe7ed6ac4f4e343287ef0e4e69e024/test_10.py#L42-L48

def test_smell_126():
    try:
        project_10.lex("12345678")

        assert False
    except:
        pass